estate/models/estate_property.py

```diff
@@ -0,0 +1,109 @@
+from odoo import models, fields, api, exceptions
+from odoo.tools import float_compare
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate model"
+
+    name = fields.Char(required=True)
+    description = fields.Char()
+    postcode = fields.Char()
+    available_from = fields.Date(
+        copy=False, default=fields.Date.add(fields.Date.today(), months=3)
+    )
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    active = fields.Boolean(default=True)
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+    )
+    state = fields.Selection(
+        string="State",
+        selection=[
+            ("new", "New"),
+            ("offer received", "Offer Received"),
+            ("offer accepted", "Offer Accepted"),
```

Comment:
```suggestion
            ("offer_accepted", "Offer Accepted"),
```
The first member of the tuple is a technical key, prevent from putting spaces into them.